docs/coding-standards/src/frameworks-and-drivers/ui/css-styling/design-tokens.md

```
@@ -38,57 +32,28 @@ UI コンポーネントの CSS スタイリングに関する規約。
-### 検証方法
-
-- オンラインツール（WebAIM Contrast Checker等）でコントラスト比を確認
-- 新しい `--color-*` と `--color-*-contrast` の組み合わせを追加する際は必ず検証
-
-### コントラスト比検証のESLint化について
-
-ESLint化不可（色のコントラスト比は静的解析で検証困難。PRレビューで確認）
+4. コントラスト検証: `--color-danger` (#c53030) + `--color-danger-contrast` (#fff) = 約5.47:1 (基準4.5:1以上をクリア)
 
----
+**例2: WCAG AA基準を余裕を持ってクリアするための色修正**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)